src/components/Header/Settings/index.js

```diff
@@ -11,12 +13,34 @@ function Settings()
 {
     return(
         <Popover
-            content={<p style={{ color: '#a7425c' }}>Coming soon</p>}
+            content={
```

Comment:
Here you're changing functionality added in this PR. Ideally, we would squash some commits or split this into two PRs. Do you think it's worth the trouble?